src/fifo/FifoBuffer.cpp

```diff
@@ -112,7 +112,7 @@ int32_t FifoBuffer::read(void *buffer, int32_t numFrames) {
         memcpy(destination, source, numBytes);
     } else {
         // just read in one shot
-        numBytes = convertFramesToBytes(framesToRead);
+        size_t numBytes = convertFramesToBytes(framesToRead);
```

Comment:
This conflicts with some bounds checking that was recently added.
https://github.com/google/oboe/blob/master/src/fifo/FifoBuffer.cpp#L120

Reply:
I've rebased master into this branch which picks up the bounds checking changes, and resolves conflicts with master. Any chance of an approval?